api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -270,6 +270,16 @@ public void createAllOfUsBillingProject(String projectName) {
         });
   }
 
+  @Override
+  public void deleteBillingProject(String billingProject) {
+    BillingApi billingApi = billingApiProvider.get();
+    retryHandler.run(
+        (context) -> {
+          billingApi.deleteBillingProject(billingProject);
+          return null;
```

Comment:
nit: you should be able to omit the return statement.

Reply:
no - this requires a return value